<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditEatery</class>
 <widget class="QDialog" name="EditEatery">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="Filters">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>421</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Lato Black</family>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>EDIT EATERY</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>211</width>
        <height>361</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>193</width>
         <height>324</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_10">
            <item>
             <widget class="QLabel" name="Filters_5">
              <property name="font">
               <font>
                <family>Lato Black</family>
                <pointsize>11</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Eatery</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_eateryName"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="Filters_6">
            <property name="font">
             <font>
              <family>Lato Black</family>
              <pointsize>11</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Status</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_Status">
          <item>
           <property name="text">
            <string>Open</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Close</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QLabel" name="Filters_7">
            <property name="font">
             <font>
              <family>Lato Black</family>
              <pointsize>11</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Address</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_Address"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="Filters_8">
            <property name="font">
             <font>
              <family>Lato Black</family>
              <pointsize>11</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Landmarks</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_Landmarks"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="Filters_9">
            <property name="font">
             <font>
              <family>Lato Black</family>
              <pointsize>11</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Accommodations</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_Accomodations"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_3">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>40</y>
        <width>192</width>
        <height>361</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="Filters_11">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>0</y>
         <width>121</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Lato Black</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Menu</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_updateEatery">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>230</y>
         <width>192</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Update Eatery</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>140</y>
         <width>191</width>
         <height>58</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QLabel" name="Filters_17">
          <property name="font">
           <font>
            <family>Lato Black</family>
            <pointsize>10</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string> Details</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_Details"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>191</width>
         <height>85</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="Filters_16">
          <property name="font">
           <font>
            <family>Lato Black</family>
            <pointsize>10</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string> Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_MenuName">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_4">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>82</y>
         <width>91</width>
         <height>58</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="Filters_14">
          <property name="font">
           <font>
            <family>Lato Black</family>
            <pointsize>10</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string> Price</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_Price"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_5">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>82</y>
         <width>91</width>
         <height>58</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_11">
        <item>
         <widget class="QLabel" name="Filters_15">
          <property name="font">
           <font>
            <family>Lato Black</family>
            <pointsize>10</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string> Category</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_Category">
          <item>
           <property name="text">
            <string>Meals</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Drinks</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="pushButton_updateMenu">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>290</y>
         <width>192</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Edit Menu</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>192</y>
         <width>191</width>
         <height>24</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QSlider" name="horizontalSlider_Index">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_Index">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>1</width>
            <height>1</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="pushButton_done">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>320</y>
         <width>192</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Done</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_addMenu">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>260</y>
         <width>192</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Add Menu</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
